lib/double_entry/reporting.rb

```diff
@@ -78,8 +78,8 @@ class AggregateFunctionNotSupported < RuntimeError; end
     # @raise [Reporting::AggregateFunctionNotSupported] The provided function
     #   is not supported.
     #
-    def aggregate(function, account, code, range, options = {})
-      Aggregate.formatted_amount(function, account, code, range, options)
+    def aggregate(function, account, code, range, filter: nil, partner_account: nil)
```

Comment:
A while ago, when grooming this ticket, we discussed making this a breaking change by turning all the arguments into keyword arguments. It would create more work in Marketplace (and other dependent apps) to upgrade, but would be a more consistent interface for the future. That would allow us to improve the symmetry in the code here by having partner_account next to account (I feel that these two things are closely related and belong together).